scripts/ble_library.py

```diff
@@ -0,0 +1,184 @@
+#!/usr/bin/python
+"""
+SPDX-License-Identifier: Apache-2.0
+Copyright (c) 2021 STMicroelectronics.
+This script updates ble library
+"""
+
+import sys
+from pathlib import Path
+import logging
+from jinja2 import Environment, FileSystemLoader
+
+include_dir = ["stm32wb/hci"]
+
+file_list = [
+    "Middlewares/ST/STM32_WPAN/ble/core/ble_bufsize.h",
```

Comment:
What are the modifications required to this file when a new patch is done to this library?
I don't think lib update automation should bring complexity to usual development flow..